### PR TITLE
create server.js

### PR DESCRIPTION
**Pull request by za-h-ra**
create and wrote server.js file

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const userRoutes = require("./routes/users-routes");
+const db = require("./db/connection");
+const PORT = process.env.PORT || 3000;
+
+const app = express();
+
+app.use(cors());
+app.use(bodyParser.json());
+app.use(logger("dev"));
+app.use("api", userRoutes);
```
Comment (indianakuffer) [inline server.js]: needs a / before api string
app.use("/api", userRoutes);

**Review by indianakuffer — CHANGES_REQUESTED** [review]
add / to line 14

File: server.js (hunk 1 above)
Comment (za-h-ra) [reply server.js]: Fixed!

**Review by indianakuffer — APPROVED** [review]
looks good

**Pull request merged**